src/redteam/commands/generate.ts

```diff
@@ -601,7 +625,36 @@ export function redteamGenerateCommand(
     .option('--force', 'Force generation even if no changes are detected', false)
     .option('--no-progress-bar', 'Do not show progress bar')
     .option('--burp-escape-json', 'Escape quotes in Burp payloads', false)
-    .action((opts: Partial<RedteamCliGenerateOptions>): void => {
+    .action(async (opts: Partial<RedteamCliGenerateOptions>): Promise<void> => {
+      setupEnv(opts.envFile);
+
+      // Handle cloud config with target
+      if (opts.config && uuidValidate(opts.config)) {
+        if (opts.target && !uuidValidate(opts.target)) {
```

Comment:
consider unnesting this block. You could move the other validations up too and IMHO it would be cleaner

Reply:
This wasn't clear, but the nesting is good because passing a target is mutually inclusive with a cloud-based config.  I've updated the comment to better reflect this.